kernel-laws/src/main/scala/cats/kernel/laws/discipline/BoundedSemilatticeTests.scala

```diff
@@ -0,0 +1,25 @@
+package cats
+package kernel
+package laws
+package discipline
+
+import org.scalacheck.{Arbitrary, Prop}
+
+trait BoundedSemilatticeTests[A] extends CommutativeMonoidTests[A] with SemilatticeTests[A] {
+
+  def laws: BoundedSemilatticeLaws[A]
+
+  def boundedSemilattice(implicit arbA: Arbitrary[A], eqA: Eq[A]): RuleSet =
+    new RuleSet {
```

Comment:
nit: most of these `new RuleSet {` where `bases` is empty could be just `new DefaultRuleSet(` right like the one you used in `BandTests`? No big deal though, just that it could be slightly more concise.

Reply:
That doesn't work for Typeclasses with more than one parent though, no?

Reply:
I didn't realize that. disregard my comment then. Maybe we can create one that does in `discipline` later.